src/main/asciidoc/reference/mapping.adoc

```diff
@@ -173,7 +190,9 @@ Constructor arguments are mapped by name to the values in the retrieved row.
 Within the mapping framework it can be applied to constructor arguments.
 This lets you use a Spring Expression Language statement to transform a key’s value retrieved in the database before it is used to construct a domain object.
 In order to reference a column of a given row one has to use expressions like: `@Value("#root.myProperty")` where root refers to the root of the given `Row`.
-* `@Column`: Applied at the field level to describe the name of the column as it is represented in the row, allowing the name to be different than the field name of the class.
+* `@Column`: Applied at the field level to describe the name of the column as it is represented in the row, allowing the name to be different from the field name of the class.
```

Comment:
Change "allowing the name to be" to "letting the name be"

-------

Changing "allow ... to ..." to "let ..." is a general rule. The changes makes sentences shorter and simpler.